src/lib/journey/journey.interfaces.ts

```diff
@@ -40,6 +40,7 @@ export interface JourneyStoreValue {
     code: Maybe<number>;
     message: Maybe<string>;
     step: Maybe<Step>;
+    troubleshoot: Maybe<string>;
```

Comment:
Adding a special property of error objects to help inform devs of placing to help troubleshoot issues.